monai/transforms/post/array.py

```diff
@@ -361,22 +361,68 @@ class RemoveSmallObjects(Transform):
     Data should be one-hotted.
 
     Args:
-        min_size: objects smaller than this size (in pixel) are removed.
+        min_size: objects smaller than this size (in pixel or in physical scale if physical_scale is True) are removed.
```

Comment:
how about change `physical_scale` -> `in_mm3: whether the specified `min_size` is in number of voxels or volume in mm^3. otherwise we'll still have ambiguity in terms of millimeter/micrometer. it's fine to just assume the spacing is defined in millimeter

```suggestion
        min_size: objects smaller than this size (in number of voxels; or volume in mm^3 if in_mm3 is True) are removed.
```